Remotion/ObjectBinding/Web.IntegrationTests/BocReferenceValue/ReadOnlyBocReferenceValueTest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using NUnit.Framework;
+using Remotion.ObjectBinding.Web.Development.WebTesting.IntegrationTests;
+using Remotion.Web.Development.WebTesting.ControlObjects;
+using Remotion.Web.Development.WebTesting.ControlObjects.Selectors;
+using Remotion.Web.Development.WebTesting.ControlSelection;
+using Remotion.Web.Development.WebTesting.ExecutionEngine.PageObjects;
+using Remotion.Web.Development.WebTesting.FluentControlSelection;
+
+namespace Remotion.ObjectBinding.Web.IntegrationTests.BocReferenceValue
+{
+  [TestFixture]
+  public class ReadOnlyBocReferenceValueTest : IntegrationTest
+  {
+    [Test]
+    public void Validate_OnReadOnlyControl_CreatesValidationErrors ()
+    {
+      var home = Start();
+      var webButtonControlObject = home.GetControl(new LocalIDControlSelectionCommand<WebButtonControlObject>(new WebButtonSelector(), "ValidateButton"));
+
+      webButtonControlObject.Click();
+
+      var readOnly = home.ReferenceValues().GetByLocalID("PartnerField_ReadOnly");
+      var validationErrors = readOnly.GetValidationErrorsForReadOnly();
+
+      Assert.That(validationErrors, Is.EqualTo(new [] {"Localized terrible partner."}));
```

Comment:
code style: please update to use e.g. "invalid" instead of "terrible" (all tests). I don't mind goofy vales, "terrible" just feels a bit over the top towards biased to me.